apps/hpl/2.1/8proc_hpl_example.sh.md

```diff
@@ -0,0 +1,46 @@
+# High-Performance Linpack (HPL v2.1)
+
+HPL is a software package that solves a (random) dense linear system
+in double precision (64 bits) arithmetic on distributed-memory
+computers. It can thus be regarded as a portable as well as freely
+available implementation of the High Performance Computing Linpack
+Benchmark.
+
+The algorithm used by HPL can be summarized by the following keywords:
+  * Two-dimensional block-cyclic data distribution
+  * Right-looking variant of the LU factorization with row partial
+    pivoting featuring multiple look-ahead depths
+  * Recursive panel factorization with pivot search and column
+    broadcast combined
+  * Various virtual panel broadcast topologies
+  * Bandwidth reducing swap-broadcast algorithm
+  * Backward substitution with look-ahead of depth 1
+
+## TEMPLATES
+
+### 8proc_hpl_example.sh.tpl
+This is a SGE literal template script that will start an 8 process 
+job to run the HPL benchmark. 
+
+### 8proc_hpl.alces.dat
+This is the HPL configuration file that adjusts the paramaters 
+that the HPL benchmark will be run with. The PxQ variables within 
```

Comment:
Not sure it definitely applies here, but look for opportunities to style "variables" or "keywords" using the `` ` `` character. e.g. ``The `PxQ` variables``.